src/EnergyPlus/SolarShading.cc

```diff
@@ -5984,127 +5987,128 @@ namespace SolarShading {
+										RhoBlBmDifBk = InterpProfSlatAng( ProfAng, SlatAng, VarSlats, Blind( BlNum ).SolBackBeamDiffRefl );
+										RGlBmFr = POLYF( CosInc, Construct( ConstrNum ).ReflSolBeamFrontCoef );
+										TBmAllShBlSc = TBlBmBm * ( TBmBm + TDifBare * RGlBmFr * RhoBlBmDifBk / ( 1 - RGlDifFr * RhoBlDifDifBk ) ) + TBlBmDif * TDifBare / ( 1 - RGlDifFr * RhoBlDifDifBk );
+
+										//added TH 12/9/2009
+										TBmBmShBlSc = TBmBmBl; //TBmBm * TBlBmBm
+										TBmDifShBlSc = TBmAllShBlSc - TBmBmShBlSc;
 
 								} else if ( ShadeFlag == ExtScreenOn ) {
 
-									// Exterior screen on: beam-beam and diffuse transmittance of exterior beam
+										// Exterior screen on: beam-beam and diffuse transmittance of exterior beam
 
-									RScBack = SurfaceScreens( ScNum ).ReflectSolBeamFront;
-									RScDifDifBk = SurfaceScreens( ScNum ).DifReflect;
-									RGlBmFr = POLYF( CosInc, Construct( ConstrNum ).ReflSolBeamFrontCoef );
-									TBmAllShBlSc = TScBmBm * ( TBmBm + RGlBmFr * RScBack * TDifBare / ( 1 - RGlDifFr * RScDifDifBk ) ) + TScBmDif * TDifBare / ( 1 - RGlDifFr * RScDifDifBk );
+										RScBack = SurfaceScreens( ScNum ).ReflectSolBeamFront;
+										RScDifDifBk = SurfaceScreens( ScNum ).DifReflect;
+										RGlBmFr = POLYF( CosInc, Construct( ConstrNum ).ReflSolBeamFrontCoef );
+										TBmAllShBlSc = TScBmBm * ( TBmBm + RGlBmFr * RScBack * TDifBare / ( 1 - RGlDifFr * RScDifDifBk ) ) + TScBmDif * TDifBare / ( 1 - RGlDifFr * RScDifDifBk );
 
-									//added TH 12/9/2009
-									TBmBmShBlSc = TBmBmSc;
-									TBmDifShBlSc = TBmAllShBlSc - TBmBmShBlSc;
+										//added TH 12/9/2009
+										TBmBmShBlSc = TBmBmSc;
+										TBmDifShBlSc = TBmAllShBlSc - TBmBmShBlSc;
 
-								} else {
-									// Between-glass blind on: beam-beam and diffuse transmittance of exterior beam
+									} else {
+										// Between-glass blind on: beam-beam and diffuse transmittance of exterior beam
 
-									if ( NGlass == 2 ) {
-										TBmAllShBlSc = t1 * tfshBB * t2 + t1 * ( tfshBB * rf2 * rbshB + tfshBd * ( 1.0 + rfd2 * rbshd ) + rfshB * rbd1 * rfshd ) * td2;
-									} else { // NGlass = 3
-										TBmAllShBlSc = t1t2 * tfshBB * t3 + t1t2 * ( tfshBB * rf3 * rbshB + tfshBd * ( 1.0 + rfd3 * rbshd ) + rbshB * ( rbd2 * tfshd + td2 * rbd1 * td2 * tfshd ) ) * td3;
-									}
+										if ( NGlass == 2 ) {
+											TBmAllShBlSc = t1 * tfshBB * t2 + t1 * ( tfshBB * rf2 * rbshB + tfshBd * ( 1.0 + rfd2 * rbshd ) + rfshB * rbd1 * rfshd ) * td2;
+										} else { // NGlass = 3
+											TBmAllShBlSc = t1t2 * tfshBB * t3 + t1t2 * ( tfshBB * rf3 * rbshB + tfshBd * ( 1.0 + rfd3 * rbshd ) + rbshB * ( rbd2 * tfshd + td2 * rbd1 * td2 * tfshd ) ) * td3;
+										}
 
-									//added TH 12/9/2009
-									TBmBmShBlSc = TBmBmBl;
-									TBmDifShBlSc = TBmAllShBlSc - TBmBmShBlSc;
+										//added TH 12/9/2009
+										TBmBmShBlSc = TBmBmBl;
+										TBmDifShBlSc = TBmAllShBlSc - TBmBmShBlSc;
 
+									}
 								}
+
```

Comment:
What's up with the above indentation?  This little section appears to be indented incorrectly, yet the closing braces are all there?  I don't get what happened.  I'll have to open the file.

Reply:
Look at above comment.